src/czechpmdevs/buildertools/editors/object/FillSession.php

```diff
@@ -119,8 +119,11 @@ public function setBiomeAt(int $x, int $z, int $id): void {
 			return;
 		}
 
-		/** @phpstan-ignore-next-line */
-		$this->explorer->currentChunk->setBiomeId($x & 0xf, $z & 0xf, $id);
+		for($y = World::Y_MIN; $y < World::Y_MAX; $y++) {
+			/** @phpstan-ignore-next-line */
+			$this->explorer->currentChunk->setBiomeId($x & 0xf, $y & 0xf, $z & 0xf, $id); //TODO: Properly support subchunks with y axis
```

Comment:
TODO?

Reply:
If Minecraft give the ability to make different biomes depending on the $y axis the user should be able to choose which subchunk has which value. Actually it sets the biomeId on every subchunks on the same X & Z axis but not on the Y as the selection.

Reply:
So you are going to implement that feature in this pull request? You can either implement that in this PR, or submit a new issue addressing that, but after submit, the TODO comment should not be there.

Reply:
I didn't expect to do it in fact, so I can just remove the TODO comment ?

Reply:
Yep, remove it and mention that in a new issue, I will implement that later.